src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,26 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+
+    private static final DateTimeFormatter CUSTOM_PARSE_DATE_FORMATTER = DateTimeFormatter
+            .ofPattern("d MMM yyyy", Locale.ENGLISH);
+    private static final DateTimeFormatter FORMAT_DATE = DateTimeFormatter
+            .ofPattern("dd MMMM yyyy HH:mm", Locale.ENGLISH);
+    private static final ZoneOffset UKRAINE = ZoneOffset.of("+02:00");
```

Comment:
please use more informative names for the people who will read your code for the first time
```suggestion
    private static final DateTimeFormatter CUSTOM_DATE_FORMATTER = DateTimeFormatter
            .ofPattern("d MMM yyyy", Locale.ENGLISH);
    private static final DateTimeFormatter CUSTOM_DATE_TIME_FORMATTER = DateTimeFormatter
            .ofPattern("dd MMMM yyyy HH:mm", Locale.ENGLISH);
    private static final ZoneOffset UKRAINE_OFFSET = ZoneOffset.of("+02:00");
```